packages/core/addon/models/table.js

```diff
@@ -77,22 +78,63 @@ export default VisualizationBase.extend(Validations, {
   }
 });
 
+/**
+ * returns default dimension fields (show clause)
+ * @param {Object} dimension - dimension from request
+ * @returns {Object} - list of field names
+ */
+function getDefaultDimensionFields(dimension) {
+  let fields = get(dimension, 'dimension.fields');
+
+  if (isArray(fields)) {
+    return fields.filter(field => isArray(field.tags) && field.tags.includes('show')).map(field => field.name);
```

Comment:
Is `dimension` a dimension metadata object? Can we use `getFieldsForTag('show')` here?

https://github.com/yahoo/navi/blob/master/packages/data/addon/models/metadata/dimension.js#L82